_data/cryptocurrencies.yml

```
@@ -335,6 +335,13 @@ websites:
       software: Yes
       doc: https://www.luno.com/help/en/articles/1000203420-how-to-enable-twofactor-authentication
 
+    - name: Mercado Bitcoin
+      url: https://www.mercadobitcoin.com.br/
+      img: mercadobitcoin.png
+      tfa: Yes
+      software: Yes
+      doc: https://suporte.mercadobitcoin.com.br/hc/pt-br/articles/360000385152-O-que-%C3%A9-e-como-ativar-autentica%C3%A7%C3%A3o-em-2-passos-token-
```

Please change the doc link to this one:
```suggestion
      doc: https://suporte.mercadobitcoin.com.br/hc/pt-br/articles/360000385152
```